notebooks/community/CODEOWNERS

```diff
@@ -18,4 +18,5 @@
 /neo4j @benofben @htappen
 /sdk/pytorch_lightning_custom_container_training.ipynb @brianchunkang
 /tensorboard @yfang1 @wattli
+/feature_store/* @inardini
```

Comment:
Maybe this is a new change, but now there is another line for /feature_store. Could you please just add yourself to the existing entry? (I don't think /* is necessary)

Reply:
Add @inardini in the /feature_store path